flink-staging/flink-scala-shell/src/test/scala/org/apache/flink/api/scala/ScalaShellITSuite.scala

```diff
@@ -225,6 +220,51 @@ class ScalaShellITSuite extends FunSuite with Matchers with BeforeAndAfterAll {
     out.toString + stdout
   }
 
+  /**
+   * tests flink shell startup with remote cluster (starts cluster internally)
+   */
+  test("start flink scala shell with remote cluster") {
+
+    val input: String = "val els = env.fromElements(\"a\",\"b\");\n" +
+      "els.print\nError\n:q\n"
+
+    val in: BufferedReader = new BufferedReader(
+      new StringReader(
+        input + "\n"))
+    val out: StringWriter = new StringWriter
+
+    val baos: ByteArrayOutputStream = new ByteArrayOutputStream
+    val oldOut: PrintStream = System.out
+    System.setOut(new PrintStream(baos))
+    val c = cluster.getOrElse(null)
+    var args : Array[String] = null
+    if (c != null) {
+      args = Array("remote",
+        c.hostname,
+        Integer.toString(c.getLeaderRPCPort))
+    } else {
+      fail
```

Comment:
Oh, I forgot the usage of `fail` method. `fail(error message)` would be better. The error message could be the reason of failure such as "Creating cluster failed.".